### PR TITLE
New games page

### PR DESCRIPTION
**Pull request by anthonyshew**
New games page.

- Introduces Drizzle
- This is our first usage of our new <GameCard /> (I've augmented the interface and I'm not in love with it)
- RSCs, woohoo!

### CODE REVIEW

File: apps/web/src/app/RootLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                src="/media/logo.svg"
+                width={89}
+              />
+            </Link>
+          }
+          linkComponent={Link}
+          links={[
+            { href: '/', children: 'Home' },
+            { href: '/test', children: 'About' },
+            { href: '/test', children: 'Games' },
```
Comment (JacobSickels) [inline apps/web/src/app/RootLayout.tsx:40]: Are there no pages for these?

**Comment by JacobSickels** [comment]
For the most part nothing in here scares me, my big question would be why we're introducing drizzle to replace prisma?

It seems to me like this works pretty close to the same as prisma, but I wonder what problems drizzle is solving or what it does better?

**Comment by anthonyshew** [comment]
A few things:
- It's so much faster it's hilarious.
- It runs on the Edge and Prisma doesn't.
- I've never liked how Prisma is a DSL. Drizzle does all the same things but in straight TS.

File: apps/web/src/app/RootLayout.tsx (hunk 1 above)
Comment (anthonyshew) [reply apps/web/src/app/RootLayout.tsx:40]: Oof, good catch.

**Pull request merged**